docs/system/restore-3.2.md

```diff
@@ -0,0 +1,27 @@
+Restoring R3.2 Templates/Standalones To R4.0
+===========================================================
+
+Although restoring R3.2 AppVMs to R4.0 generally works with no problem, there are additional steps that need to be performed when doing the same with Templates and Standalone VMs.
+If these steps are not done, the result is an inability to start programs in the restored VM.
+
+1. After restoring your Template or StandaloneVM, go to Qube Settings -> Basic tab.
+   Confirm Networking is set to one valid for your R4.0 system.
+   Then, under the Advanced tab, confirm Virtualization Mode is `PVH`, and that Kernel is set to `default`.
+2. From a dom0 prompt, enter `qvm-run <templatename> gnome-terminal`.
```

Comment:
'xterm' is the least common denominator  - use it instead of `gnome-terminal` ?

or,  "From a dom0 prompt, enter `qvm-run <templatename> gnome-terminal` or `qvm-run <templatename> xterm`